Mage/src/main/java/mage/game/match/MatchOptions.java

```diff
@@ -288,4 +291,19 @@ public MulliganType getMulliganType() {
         return mulliganType;
     }
 
+    public Collection<DeckCardInfo> getPerPlayerEmblemCards() {
+        return perPlayerEmblemCards;
+    }
+
+    public void setPerPlayerEmblemCards(Collection<DeckCardInfo> perPlayerEmblemCards) {
```

Comment:
curious, why storing the set directly? would it make more sense to have a final set internal, and have this method clear it and add all elements passed to it? or does it just not matter in this context?

Reply:
If it matters either way, I wouldn't know why. In this case, it stores a `Collection` because that's the least specific interface with all the functionality needed, as far as I know, and that seemed like the sensible type for the field.